Src/library/memory_analyzer/object_graph/ma_figure_factory.e

```diff
@@ -60,17 +62,19 @@ feature -- Basic operations
 --					end
 --				end
 --			end
+			to_implement (generator + ".model_from_xml: Not yet implemented!")
+			;(create {EXCEPTIONS}).die (-1)
```

Comment:
If no void-safety is required, non-implemented features are either implemented with require/check/ensure False.